vm-setup/roles/v1a2_integration_test/tasks/provision.yml

```
@@ -0,0 +1,104 @@
+---
+# Populate crs with environment variables
+  - name: Template cluster.yaml
+    template:
+      src: "{{ HOME }}/metal3-dev-env/vm-setup/roles/v1a2_integration_test/crs/cluster.yaml"
+      dest: "/tmp/cluster.yaml"
+    ignore_errors: true
```

why ignoring errors ?